# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1327455190 -3600
#      Wed Jan 25 02:33:10 2012 +0100
# Branch gui
# Node ID 2254d7c14f418be7caf8972db2762bb24b668223
# Parent  6e6194a7b102f32be474fedb49ecef6c5ab6037b
Provided a patch in the pro for Windows, which does not need qterminal.

* octave-gui.pro: Embraced qterminal library with unix tags.

diff --git a/gui/octave-gui.pro b/gui/octave-gui.pro
--- a/gui/octave-gui.pro
+++ b/gui/octave-gui.pro
@@ -26,17 +26,17 @@ MOC_DIR             = moc-files         
 OBJECTS_DIR         = object-files                  # Folder for object files
 
 TRANSLATIONS        += languages/generic.ts \
                        languages/de-de.ts \
                        languages/pt-br.ts \
                        languages/es-es.ts \
                        languages/ru-ru.ts \
                        languages/uk-ua.ts           # Available translations
-LIBS                += -lqscintilla2 -Lqterminal/libqterminal -lqterminal $$system(mkoctfile -p LIBS) $$system(mkoctfile -p OCTAVE_LIBS)
+LIBS                += -lqscintilla2  $$system(mkoctfile -p LIBS) $$system(mkoctfile -p OCTAVE_LIBS)
 
 mac {
     CONFIG -= app_bundle
 }
 
 # Includepaths and libraries to link against:
 INCLUDEPATH         += src src/terminal src/qirc src/backend qterminal/libqterminal \
                        $$system(mkoctfile -p INCFLAGS)
@@ -47,17 +47,17 @@ mac {
 
 QMAKE_LIBDIR        += $$system(octave-config -p OCTLIBDIR)
 
 mac {
     LFLAGS += -L/opt/local-native/lib
 }
 
 unix {
-    LIBS +=  -lutil
+    LIBS +=  -lutil -Lqterminal/libqterminal -lqterminal
 }
 
 win32-g++ {
     QMAKE_LFLAGS += --enable-auto-import
 }
 
 win32-msvc* {
     DEFINES += QSCINTILLA_DLL
diff --git a/gui/src/MainWindow.cpp b/gui/src/MainWindow.cpp
--- a/gui/src/MainWindow.cpp
+++ b/gui/src/MainWindow.cpp
@@ -313,17 +313,17 @@ MainWindow::construct ()
   m_documentationWidgetSubWindow
       ->setWindowIcon (ResourceManager::instance ()->icon (ResourceManager::Documentation));
   m_documentationWidgetSubWindow->setFocusProxy (m_documentationWidget);
   m_documentationWidgetSubWindow->setStatusTip (tr ("Browse the Octave documentation for help."));
   m_documentationWidgetSubWindow->setMinimumSize (300, 300);
 
   // Chat subwindow.
   m_ircWidgetSubWindow = new NonClosableMdiSubWindow (this);
-  m_ircWidgetSubWindow->setWidget (m_ircWidget);
+  m_ircWidgetSubWindow->setWidget(m_ircWidget);
   m_centralMdiArea->addSubWindow (m_ircWidgetSubWindow, Qt::WindowTitleHint | Qt::WindowMinMaxButtonsHint);
 
   m_ircWidgetSubWindow->setObjectName ("ChatWidgetSubWindow");
   m_ircWidgetSubWindow->setWindowTitle (tr ("Chat"));
   m_ircWidgetSubWindow
       ->setWindowIcon (ResourceManager::instance ()->icon (ResourceManager::Chat));
   m_ircWidgetSubWindow->setStatusTip(tr ("Instantly chat with other Octave users for help."));
   m_ircWidgetSubWindow->setFocusProxy (m_ircWidget);
